intelmq/bots/parsers/shadowserver/config.py

```diff
@@ -928,3 +936,303 @@ def validate_ip(value):
         'classification.identifier': 'opennatpmp',
     },
 }
+
+# https://www.shadowserver.org/wiki/pmwiki.php/Services/Compromised-Website
+compromised_website = {
+    'required_fields': [
+        ('time.source', 'timestamp', add_UTC_to_timestamp),
+        ('source.ip', 'ip'),
+        ('source.port', 'port')
+    ],
+    'optional_fields': [
+        ('source.reverse_dns', 'hostname'),
+        ('malware.name', 'tag'),
+        ('protocol.application', 'application'),
+        ('source.asn', 'asn'),
+        ('source.geolocation.cc', 'geo'),
+        ('source.geolocation.region', 'region'),
+        ('source.geolocation.city', 'city'),
+        ('event_description.url', 'url'),
+        ('event_description.target', 'http_host'),
+        ('event_description.text', 'category'),
```

Comment:
Shadowserver defines this as 
> Type of maliciousness the compromised website is being used for 

Do you know which values are possible here (spam, phishing, ...) ? Might the `classification.type` get dynamic in this case?